src/base/AddMovie/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import styled from './style.module.scss'
+
+function index() {
+  return (
+    <button className={`${styled.addMovie} p-2`} type="button">
+      + ADD MOVIE
```

Comment:
make it more generic, without static text so you can reuse it

Reply:
Built dist dir removed and added in gitignore
Got it, so I use only one css_modules I thought I could use others as well
Do you mean there's need to change static text "+ ADD MOVIE" like this one?!
Okay, i'll do this one as generic then

Reply:
done